lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -310,6 +333,59 @@ def get_driver_options(driver_options):
 
 class DockerNetworkManager(object):
 
+    def _get_minimal_versions(self):
+        # TODO: Move this and the same from docker_container.py to docker_common.py
+        self.option_minimal_versions = dict()
+        for option, data in self.client.module.argument_spec.items():
+            self.option_minimal_versions[option] = dict()
+        self.option_minimal_versions.update(dict(
+            scope=dict(docker_py_version='2.6.0', docker_api_version='1.29'),
+            attachable=dict(docker_py_version='2.0.0', docker_api_version='1.25'),
```

Comment:
I couldn't find this in the [1.25 API documentation](https://docs.docker.com/engine/api/v1.25/#operation/NetworkCreate), but only in the [1.26 API documentation](https://docs.docker.com/engine/api/v1.26/#operation/NetworkCreate). Interestingly, the docker-py source code claims it has been supported since 1.24...

Reply:
https://docs.docker.com/engine/reference/commandline/network_create/ claims it's available on 1.25+. I'll set the minimum to 1.26 just to be safe.

Reply:
Hmm, that's pretty inconsistent. So now we got three different choices... :) I guess it's fine to leave it with 1.25.